src/Microsoft.IdentityModel.Tokens/Validation/Results/Details/TokenTypeValidationError.cs

```diff
@@ -15,8 +15,10 @@ internal TokenTypeValidationError(
             MessageDetail messageDetail,
             Type exceptionType,
             StackFrame stackFrame,
-            string? invalidTokenType)
-            : base(messageDetail, ValidationFailureType.TokenTypeValidationFailed, exceptionType, stackFrame)
+            string? invalidTokenType,
+            ValidationFailureType? validationFailureType = null,
```

Comment:
Not understanding why validationFailureType should ever be null.

Reply:
Removed the nullability.